proto/rill/admin/v1/api.proto

```diff
@@ -349,6 +349,10 @@ service AdminService {
     option (google.api.http) = {get: "/v1/github/user"};
   }
 
+  rpc ListGithubUserRepos(ListGithubUserReposRequest) returns (ListGithubUserReposResponse) {
+    option (google.api.http) = {get: "/v1/github/repositories/list"};
```

Comment:
I think list can be skipped and the path can be changed to `/v1/github/repositories` to reflect that the repositories are for a user.

Reply:
`GetGithubRepoStatus` has the endpoint `/v1/github/repositories` already. Perhaps we should move that to `v1/github/repositories/status`

Reply:
Sorry I missed the `user` part. It should be `/v1/github/user/repositories` IMO